View/ViewHandler.php

```diff
@@ -254,18 +258,26 @@ protected function getSerializationContext(View $view)
     {
         $context = $view->getSerializationContext();
 
-        if ($context->attributes->get('groups')->isEmpty() && $this->exclusionStrategyGroups) {
+        if($context instanceof Context) {
+            $JMSAdapter = new JMSContextAdapter();
+            $context = $JMSAdapter->transform($context);
+        }
+
+        if (empty($context->getGroups()) && $this->exclusionStrategyGroups) {
```

Comment:
passing the return value of a function to ``empty()`` works only on PHP 5.5+. AFAIK, this bundle still supports older versions